examples/scenes/src/simple_text.rs

```diff
@@ -31,12 +34,75 @@ const FONT_DATA: &[u8] = include_bytes!("../../assets/roboto/Roboto-Regular.ttf"
 
 pub struct SimpleText {
     gcx: GlyphContext,
+    font: Font,
 }
 
 impl SimpleText {
     pub fn new() -> Self {
         Self {
             gcx: GlyphContext::new(),
+            font: Font::new(Blob::new(Arc::new(FONT_DATA)), 0),
+        }
+    }
+
+    pub fn add_run<'b>(
+        &mut self,
+        builder: &mut SceneBuilder,
+        size: f32,
+        brush: impl Into<BrushRef<'b>>,
+        transform: Affine,
+        text: &str,
+    ) {
+        let font = FontRef {
+            data: FONT_DATA,
+            offset: 0,
+        };
+        let brush = brush.into();
+        if let Some(cmap) = font.cmap() {
+            if let Some(hmtx) = font.hmtx() {
+                let upem = font.head().map(|head| head.units_per_em()).unwrap_or(1000) as f64;
+                let scale = size as f64 / upem;
+                let hmetrics = hmtx.hmetrics();
+                let default_advance = hmetrics
+                    .get(hmetrics.len().saturating_sub(1))
+                    .map(|h| h.advance_width)
+                    .unwrap_or(0);
+                let mut pen_x = 0f64;
+                builder
+                    .draw_glyphs(&self)
+                    .font_size(size)
+                    .transform(transform)
+                    .glyph_transform(Some(Affine::new([
+                        1.,
+                        0.,
+                        20f64.to_radians().tan(),
```

Comment:
This could do with a comment; I'm struggling to understand the context.

Unless it's just a testing change to make all text faux-italicised?

Reply:
Yes, it’s just a manual skew to test that the per glyph transform works correctly. When I get a chance, I’ll write up a patch to add this to kurbo. In the meantime, I’ll tack a comment on here.

Reply:
Oh, I didn't realize this was in `simple_text.rs` and not in the test scene! Good catch. I'll shuffle it around.